### ISSUE TITLE
load crashes if parameter file has commented out lines starting with #

### ISSUE THREAD

**Issue by peeples** [yt-project/yt#1584]
### Bug report

**Bug summary**

If the parameter file has a line started with #, it can't be loaded

**Code for reproduction**

```python
import yt
ds = yt.load('RD0027/RD0027')
```

**Actual outcome**

```
ValueError                                Traceback (most recent call last)
<ipython-input-2-d4d26e2f34da> in <module>()
----> 1 ds = yt.load("RD0027/RD0027")
/Users/molly/anaconda2/envs/astroconda/lib/python2.7/site-packages/yt/convenience.pyc in load(*args, **kwargs)
     84     candidates = find_lowest_subclasses(candidates)
     85     if len(candidates) == 1:
---> 86         return candidates[0](*args, **kwargs)
     87     if len(candidates) == 0:
     88         if ytcfg.get("yt", "enzo_db") != '' \
/Users/molly/anaconda2/envs/astroconda/lib/python2.7/site-packages/yt/frontends/enzo/data_structures.pyc in __init__(self, filename, dataset_type, file_style, parameter_override, conversion_override, storage_filename, units_override, unit_system)
    680         self.storage_filename = storage_filename
    681         Dataset.__init__(self, filename, dataset_type, file_style=file_style,
--> 682                          units_override=units_override, unit_system=unit_system)
    683 
    684     def _setup_1d(self):
/Users/molly/anaconda2/envs/astroconda/lib/python2.7/site-packages/yt/data_objects/static_output.pyc in __init__(self, filename, dataset_type, file_style, units_override, unit_system)
    304         self._create_unit_registry()
    305 
--> 306         self._parse_parameter_file()
    307         self.set_units()
    308         self._assign_unit_system(unit_system)
/Users/molly/anaconda2/envs/astroconda/lib/python2.7/site-packages/yt/frontends/enzo/data_structures.pyc in _parse_parameter_file(self)
    741                 self._parse_enzo3_parameter_file(f)
    742             else:
--> 743                 self._parse_enzo2_parameter_file(f)
    744 
    745     def _parse_enzo3_parameter_file(self, f):
/Users/molly/anaconda2/envs/astroconda/lib/python2.7/site-packages/yt/frontends/enzo/data_structures.pyc in _parse_enzo2_parameter_file(self, f)
    783         for line in (l.strip() for l in f):
    784             if len(line) < 2: continue
--> 785             param, vals = (i.strip() for i in line.split("=",1))
    786             # First we try to decipher what type of value it is.
    787             vals = vals.split()
ValueError: need more than 1 value to unpack
```

**Expected outcome**

That line should be ignored.

**Version Information**

  * Operating System:
  * Python Version: 2.7
  * yt version: 3.4.0
  * Other Libraries (if applicable): 

conda

**Comment by ngoldbaum** [comment]
In particular, adding a line like this to the middle of `IsolatedGalaxy/galaxy0030/galaxy0030`:

```
### comment!
```

Is sufficient to trigger the crash.

**Comment by JBorrow** [comment]
See MR #1586

**Comment by brittonsmith** [comment]
This looks like it's due to a lack of an equal sign and not because a line starts with `#`.  There are multiple lines in the Enzo parameter files that start with `#`.  I don't think we want to ignore them.

**Comment by JBorrow** [comment]
Can this be closed now? @peeples

**Comment by ngoldbaum** [comment]
Ah yes, this was fixed. @JBorrow in your commit message you said "Fixes issue #1584". Github only autocloses the issue if you do "Closes #1584". No worries though - just for future reference!

**Comment by JBorrow** [comment]
Thanks @ngoldbaum, I had never closed through a commit before :)
